### ISSUE TITLE
Keyboard navigation difficult within Settings editor > keyboard shortcuts

### ISSUE THREAD

**Issue by e218736** [jupyterlab/jupyterlab#15546]
### Problem

Navigating keyboard shortcuts in settings editor using keyboard has limited functionality compared with using a mouse.

For example I am unable to interact with the shortcut keys div to enable editing the keys within the input field.

In addition, inability to focus on elements within each row using keyboard results in not being able to see or select the option to add a secondary keyboard shortcut.

### Proposed Solution

Converting interactive divs to buttons to ensure tab-ability for keyboard users. 
CSS styling of focus events to replicate mouse hover functionality for keyboard users.

### Additional context

Discovered this whilst working on a solution to [Settings editor keyboard shortcut punctuation issues](https://github.com/jupyterlab/jupyterlab/issues/15497) so will update if other navigation issues arise

**Comment by JasonWeill** [comment]
@e218736 Thanks for opening this! Just to be certain, which version of JupyterLab are you using? The settings editor is very different in Lab 3 compared with Lab 4, for example.

**Comment by e218736** [comment]
@JasonWeill This is using Version 4.1.0a4

**Comment by IsabelParedes** [comment]
Hi! I also found it difficult navigating the settings editor with the keyboard only. I am unable to navigate to any of the settings on the left panel using the keyboard only. Even after clicking one of the tabs on the left panel (`CodeMirror` in the video), I still cannot navigate to any of the other tabs.

This test was using JupyterLab version 4.5.2

https://github.com/user-attachments/assets/ffaad30d-9a8a-4f99-993d-604d503387c6
